environments/prod/prod.tfvars

```diff
@@ -1773,6 +1773,14 @@ frontends = [
     ssl_mode         = "AzureKeyVault"
     certificate_name = "courttribunalfinder-service-gov-uk"
   },
+  {
+      product          = "rpts"
+      name             = "rpts"
+      mode             = "Detection"
+      custom_domain    = "www.residential-property-tribunals-service.gov.uk"
+      backend_domain   = ["firewall-prod-int-palo-prod.uksouth.cloudapp.azure.com"]
+      certificate_name = "residential-property-tribunals-service-gov-uk-platform-hmcts-net"
```

Comment:
is this right? I would have expected the name to be:
`residential-property-tribunals-service-gov-uk`

Reply:
Yep, I checked in the vault and came through as this, there's others with similar ending in the file so thought I'd try to change @timja